spec/dfe/analytics_spec.rb

```diff
@@ -1,187 +1,183 @@
-# frozen_string_literal: true
-
-RSpec.describe DfE::Analytics do
-  it 'has a version number' do
-    expect(DfE::Analytics::VERSION).not_to be nil
+RSpec.describe 'Analytics flow', type: :request do
```

Comment:
Not too sure I understand all the changes in this spec. Were the tests rearranged  ? Difficult to follow the changes.

Reply:
Not sure what happened here - these changes were not intended!